data/presets/presets/lounger.json

```diff
@@ -0,0 +1,32 @@
+{
+    "icon": "temaki-bench",
+    "fields": [
+        "backrest",
```

Comment:
@quincylvania @animesh-007 this specific field sounds like something that should not be included, right? I would not know what a lounger without backrest looks like.

Reply:
I changed some of the fields, see https://github.com/openstreetmap/iD/commit/1c3928160f68734aedadf977b723c833e8b19791.